go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/jaypipes/ghw
 
-go 1.19
+go 1.21
```

Comment:
do we need any feature of go >= 1.20, or do one of our uplifted deps?

Reply:
@ffromani no, not particularly. I was just keeping us up to date with more modern Go :)

Reply:
I'm a fan of keeping backward compatibility as much as possible for library code, but I don't have any real objection here so let's bump it.
Perhaps next time let's try to be a tad more conservative for the sake of making life of the consumers of ghw easier